clients/core-java/src/main/java/org/commonjava/indy/client/core/Indy.java

```diff
@@ -267,6 +269,18 @@ private void setup( IndyClientModule module )
         }
     }
 
+    // DA, Builder, Orchestrator, etc. If available, this will be sent as a request header.
+    public void setComponentId( String componentId )
+    {
+        http.addDefaultHeader( HEADER_COMPONENT_ID, componentId );
+    }
+
+    // Default headers will be sent along with each request
+    public void setDefaultHeader( String key, String value )
```

Comment:
Can we make these method names consistent? I think set vs add are slightly different implied semantics. We might want to implement a true 'set' later which would maintain a single header value for the key...also, do we need a remove method? Maybe not...

Reply:
Using add is better here. will update.